app/coffee/PersistenceManager.coffee

```diff
@@ -65,6 +66,7 @@ module.exports = PersistenceManager =
 				sendImmediately: true
 			jar: false
 			timeout: MAX_HTTP_REQUEST_LENGTH
+			maxAttempts: 2  # for requestretry
```

Comment:
Ditto as for above.

There's a handy requestretry().defaults({....}) method which could save some duplication here.
https://github.com/FGRibreau/node-request-retry#modifying-request-options

It creates a wrapper, so doesn't modify the global object.